torch/testing/_internal/common_modules.py

```diff
@@ -1399,6 +1399,15 @@ def module_inputs_torch_nn_LSTM(module_info, device, dtype, requires_grad, train
                skips=(
                    DecorateInfo(skipIfMps, 'TestModule', dtypes=[torch.float64]),),
                supports_gradgrad=False),
+    # TransformerEncoder takes the same inputs as TransformerEncoderLayer
+    ModuleInfo(torch.nn.TransformerEncoder,
+               train_and_eval_differ=True,
+               module_inputs_func=module_inputs_torch_nn_TransformerEncoderLayer,
```

Comment:
Note that using the same `module_inputs_func` for `TransformerEncoder` won't work since `TransformerEncoderLayer` and `TransformerEncoder` are constructed differently. The `module_inputs_func` specifies both constructor args and forward pass args.

Reply:
Thanks!